### ISSUE TITLE
Replace git-like short-hands with `git`

### ISSUE THREAD

**Issue by jackorp** [pvalena/theprototype#1]
TODO:
 - [ ] create-pr.sh : unknown git alias

_ _ _ _
Originally:
_This line: https://github.com/pvalena/theprototype/blob/main/fedora/create_pr.sh#L57
contains `gith`, I am not sure what that one should be._

_Additionally, https://github.com/pvalena/theprototype/blob/main/fedora/create_pr.sh#L81
contains `gitb` (which seems like `git branch`, at least it works for me like that)._

**Comment by pvalena** [comment]
Yes, this is from my script /wrapper set which I've not open-sourced yet.

Let's fix these this tooling-wide.
